src/Common/src/CoreLib/System/Numerics/Vector.cs

```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 #nullable enable
-#if netcoreapp
+#if netcoreapp || netcoreapp30
```

Comment:
Is this actually compiled into a netcoreapp30 assembly?  If so which one?  I couldn't see that.  Same comment applies for all of the changes under Common/src/CoreLib

Reply:
Same comment applies to all these modified IFDEFS.  Make sure we are actually building and shipping the source in an assembly that would use the IFDEF.